Tasks/PythonToolInstaller/AzureStorageArtifacts/AzureStorageArtifactDownloader.ts

```diff
@@ -0,0 +1,80 @@
+import * as Q from 'q';
```

Comment:
This was copied from somewhere else, correct?  Should we put a comment at the top re: where it was copied?

I'm not sure how we should go about keeping files like this in sync.  We may want to reconsider how to handle files like this in the future if 'common' doesn't work.

Reply:
This is copied in a few of our tasks right now.  I was going to try to move it and update all those tasks in a separate change.  If common can't hold it (because modules in common can't depend on other modules in common, apparently) we should consider moving it out to another repo.  Maybe task lib or tool lib could take it.